lib/fog/openstack/models/volume/volume.rb

```diff
@@ -6,8 +6,10 @@ class OpenStack
       class Volume < Fog::OpenStack::Model
         identity :id
 
-        attribute :display_name,        :aliases => 'displayName'
-        attribute :display_description, :aliases => 'displayDescription'
+        # NOTE: The attributes "name" and "description" are called
+        # "display_name" and "display_description" in API version v1.
+        attribute :name,                :aliases => ['display_name', 'displayName', :display_name]
+        attribute :description,         :aliases => ['display_description', 'displayDescription', :display_description]
```

Comment:
`:aliases` are commonly given as strings, but in here, I also needed to give it as a symbol. If I don't, loading of objects via e.g. `@service.volumes.all(...)` fails when calling `object.merge_attributes({ 'display_name' => 'foo', ... })`. I'm not sure if this is a problem with fog-core (should either strings or symbols be normalized at some point inside `Fog::Attributes`?) or if I'm just the first to handle a renaming of attribute names in an API? (If there is a way to do this right, pointers are highly appreciated.)